src/DurableSDK/DurableTaskHandler.cs

```diff
@@ -83,7 +85,7 @@ internal class DurableTaskHandler
                                         retryOptions.MaxNumberOfAttempts,
                                         onSuccess:
                                             result => {
-                                                output(TypeExtensions.ConvertFromJson(result));
```

Comment:
Throughout this file, we remove the dependency from `TypeExtensions`. This allows us to remove the tight-dependency between the `DurableSDK` component, and the worker.  We want to remove this dependency because want to set a foundation where the DF SDK in PowerShell can be de-coupled from the worker-specific logic.